Serverp/src/Server.java

```diff
@@ -85,21 +109,68 @@ private static void SetUpConnections()
 	}
 
 	/*
-	 * Runnable class to handle instances of clients that are connected
+	 * Writes a String to a client through socket socket
+	 */
+	private static void writeToCLient(String string, Socket socket) throws IOException
+	{
```

Comment:
Is this method necessary? I don't believe we have any situations in which we will be sending a string to the client.